django_auth_adfs/backend.py

```diff
@@ -161,7 +161,8 @@ def create_user(self, claims):
             user.set_unusable_password()
         return user
 
-    def update_user_attributes(self, user, claims):
+    # https://github.com/snok/django-auth-adfs/issues/241
+    def update_user_attributes(self, user, claims, mapped_fields=settings.CLAIM_MAPPING):
```

Comment:
What do you think about `field_claim_map` or `claim_mapping` rather than `mapped_fields`? I think it should be closer to the setting name.

Reply:
Done: https://github.com/snok/django-auth-adfs/pull/242/commits/91f7ee5273cacda41e7a790e378bd8698fc3f39a